### ISSUE TITLE
Provide Variational Approximator functionality

### ISSUE THREAD

**Issue by danielkelshaw** [danielkelshaw/WeightUncertainty#11]
At the moment there is no way to calculate the loss from the Bayesian Linear layers - it would be a good idea to provide a `variational_approximator` wrapper which adds additional functionality to `nn.Module` to allow for simple calculation of the `ELBO` loss.
